### PR TITLE
blog: add postgREST aggregate functions

### PR DESCRIPTION
**Pull request by steve-chavez**
Adds blog post for PostgREST Aggregate Functions

https://zone-www-dot-com-git-fork-steve-chavez-tim-blog-post-supabase.vercel.app/blog

### CODE REVIEW

**Review by awalias — CHANGES_REQUESTED** [review]
looks good I think we just need a thumbnail and also to run the script that adds it to the RSS file

File: apps/www/lib/authors.json [hunk 1]
```diff
@@ -421,5 +421,13 @@
...
     "author_url": "https://github.com/imor",
     "author_image_url": "https://github.com/imor.png"
+  },
+  {
+    "author_id": "timabdulla",
+    "author": "Tim Abdulla",
+    "username": "timabdulla",
+    "position": "Guest Author",
+    "author_url": "https://github.com/timabdulla",
+    "author_image_url": "https://github.com/timabdulla.png"
```
Comment (awalias) [inline apps/www/lib/authors.json:431]: json parsing struggling here for some reason https://vercel.com/supabase/zone-www-dot-com/GUk92NyPsmEsheqspFLJVbbdHJsL

Comment (jordienr) [reply apps/www/lib/authors.json:431]: CleanShot 2024-03-01 at 00 05 30@2x
looks like vercel has a trailing comma in the source but I checked out the branch locally and I don't see it so I cant fix it :/

**Review by awalias — APPROVED** [review]
good to go

**Pull request merged**